src/libumf.c

```diff
@@ -21,6 +24,11 @@ static unsigned long long umfRefCount = 0;
 
 int umfInit(void) {
     if (util_fetch_and_add64(&umfRefCount, 1) == 0) {
+        if (util_env_var_has_str("UMF_VERSION", "1")) {
+            int umf_ver = umfGetCurrentVersion();
+            printf("UMF_VERSION: %i.%i\n", UMF_MAJOR_VERSION(umf_ver),
```

Comment:
why we can't use our logging mechanism here? please do not use printf.

Reply:
Done: https://github.com/oneapi-src/unified-memory-framework/pull/552